cfg/conf.sample.php

```diff
@@ -146,6 +151,17 @@
 syntaxhighlighting = "Source Code"
 markdown = "Markdown"
 
+[available_templates]
+; List of available for selection templates when "templateselection" option is enabled
+bootstrap
+bootstrap-page
+bootstrap-dark
+bootstrap-dark-page
+bootstrap-compact
+bootstrap-compact-page
```

Comment:
I think we can even create two additional combinations with dark in the mix:
```suggestion
bootstrap-compact-page
bootstrap-dark-compact
bootstrap-dark-compact-page
```

Reply:
Unfortunately we can not create the combinations you suggested. It can not be at the same time *-compact* and *-dark mode because both of them have to start from 9 character of the template name string
https://github.com/PrivateBin/PrivateBin/blob/0eef8d62d719fe0cf45210bbf056eb4a5e51f750/tpl/bootstrap.php#L3-L5